src/styles/Shared.css

```diff
@@ -381,6 +385,10 @@ button.btn-link:hover .logout-icon {
   margin-top: 0 !important;
 }
 
+.mt-md {
```

Comment:
I like this approach because we don't have a lot of "smart" components, css isolation and at the same time a lot of duplicated plain Html with not very clear classes like "Exchange-something".

I think we can make some kind of custom tailwind, with basic classes for flex boxes, indents, etc.